src/main/java/org/prebid/server/bidder/ix/IxBidder.java

```diff
@@ -132,8 +132,8 @@ private static Site modifySite(Site site, ExtImpIx extImpIx) {
         return site == null
                 ? null
                 : site.toBuilder()
-                        .publisher(modifyPublisher(site.getPublisher(), extImpIx.getSiteId()))
```

Comment:
Indentation removal was accidendal or purposely?

Reply:
Seems this is default IDE formatter. Gues we my talk offline on it.